socorro/unittest/cron/jobs/test_monitoring.py

```diff
@@ -157,65 +157,82 @@ def test_get_javascript_vulnerabilities_failure(self, mock_popen, app_config):
     def test_get_javascript_vulnerabilities_found(self, mock_popen, app_config):
         app = self.get_app(app_config)
 
-        # Adapated from nsp output for a jquery issue
-        output = json.dumps([
-            {
-                'id': 328,
-                'updated_at': '2017-04-20T04:19:42.040Z',
-                'created_at': '2017-03-20T21:50:28.000Z',
-                'publish_date': '2017-03-21T18:23:53.000Z',
-                'overview': 'This is an error',
-                'recommendation': 'Upgrade to v3.0.0 or greater.',
-                'cvss_vector': 'CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:L/I:L/A:N',
-                'cvss_score': 7.2,
-                'module': 'mylibrary',
-                'version': '0.9.0',
-                'vulnerable_versions': '<1.0.0',
-                'patched_versions': '>=3.0.0',
-                'title': 'Cross-Site Scripting (XSS)',
-                'path': [
-                    'socorro-webapp-django@0.0.0',
-                    'jquery@2.1.0',
-                ],
-                'advisory': 'https://nodesecurity.io/advisories/328',
+        # Adapated from npm audit output for a jest issue
+        output = json.dumps({
+            "actions": [
+                # Skipping actions because we don't do anything with them.
+            ],
+            "advisories": {
```

Comment:
The shape of the output is different than it used to be. I tried to map like things so the `Vulnerability` instance was similar.